--- ./filter.orig
+++ /etc/raddb/policy.d/filter
@@ -61,12 +61,12 @@
 		#  must have at least 1 string-dot-string after @
 		#  e.g. "user@site.com"
 		#
-		if ((&User-Name =~ /@/) && (&User-Name !~ /@(.+)\.(.+)$/))  {
-			update request {
-				&Module-Failure-Message += 'Rejected: Realm does not have at least one dot separator'
-			}
-			reject
-		}
+#		if ((&User-Name =~ /@/) && (&User-Name !~ /@(.+)\.(.+)$/))  {
+#			update request {
+#				&Module-Failure-Message += 'Rejected: Realm does not have at least one dot separator'
+#			}
+#			reject
+#		}
 
 		#
 		#  Realm ends with a dot
