--- /etc/raddb/sites-available/check-eap-tls.orig
+++ /etc/raddb/sites-available/check-eap-tls
@@ -40,10 +40,31 @@
 	#
 	# By default, we just accept the request:
 	#
-	update config {
-		&Auth-Type := Accept
-	}
+#	update config {
+#		&Auth-Type := Accept
+#	}
 
+	# At this place the certificate has been checked for validity
+	# The common name from the certificate is set as user name
+
+        update {
+                &request:User-Name := TLS-Client-Cert-Common-Name
+        }
+
+	# Check if the user name (common name) is available in the users (authorize) file
+        files
+
+	# Reject if the user name is not in the users file
+        if (ok) {
+                update config {
+                    &Auth-Type := Accept
+                }
+        }
+        else {
+                update config {
+                    &Auth-Type := Reject
+             }
+        }
 
 	#
 	# Check the client certificate matches a string, and reject otherwise
